backend/src/modules/columns/controller/columns.controller.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { EventEmitterModule } from '@nestjs/event-emitter';
+import { getModelToken } from '@nestjs/mongoose';
+import { SchedulerRegistry } from '@nestjs/schedule';
+import { Test } from '@nestjs/testing';
+import {
+	boardRepository,
+	getBoardApplication,
+	getBoardService,
+	updateBoardApplication,
+	updateBoardService
+} from 'src/modules/boards/boards.providers';
+import { deleteCardService, getCardService } from 'src/modules/cards/cards.providers';
+import * as CommunicationsType from 'src/modules/communication/interfaces/types';
+import SocketGateway from 'src/modules/socket/gateway/socket.gateway';
+import {
+	getTeamApplication,
+	getTeamService,
+	teamRepository,
+	teamUserRepository
+} from 'src/modules/teams/providers';
+import { deleteVoteService } from 'src/modules/votes/votes.providers';
+import {
+	columnRepository,
+	updateColumnApplication,
+	updateColumnService
+} from '../columns.providers';
+import ColumnsController from './columns.controller';
+
+describe('ColumnsController', () => {
+	let controller: ColumnsController;
+
+	beforeEach(async () => {
```

Comment:
```suggestion
	beforeAll(async () => {
```
Instead of doing this in every test, this only have to be done once